boards/posix/nrf52_bsim/common/phy_sync_ctrl.c

```diff
@@ -23,7 +23,7 @@
 
 static struct {
 	double start_offset;
```

Comment:
Should `start_offset` also be changed?

Reply:
Nope. start_offset is used as a double all the way thru.